service/src/utils/cli_parsers/parseRegisterArgs.ts

```diff
@@ -0,0 +1,95 @@
+import { ethers } from "ethers";
+import { exit } from "process";
+import { Web3Interface } from "scripts/web3-interface";
+import { archLogger } from "../chalk-theme";
+import { CLI_BAD_REGISTER_PROFILE_ARG } from "../exit-codes";
+
+const diggingFee = 'digging-fee';
+const rewrapInterval = 'rewrap-interval';
+const freeBond = 'free-bond';
```

Comment:
not sure if this is an improvement, but these could be defined as an enum that is shared between register / update args scripts-

```
enum ProfileArgNames {
  DIGGING_FEE = 'digging-fee',
  REWRAP_INTERVAL = 'rewrap-interval',
  FREE_BOND = 'free-bond'
}

...
 switch (argName) {
            case ProfileArgs.DIGGING_FEE:
```